reloader/reloader_test.go

```diff
@@ -140,6 +131,17 @@ func TestReload(t *testing.T) {
 	}
 }
 
+/// existingFile implements fs.FileInfo / os.FileInfo and returns information
+/// as if it were a basic file that existed. This is used to mock out os.Stat.
```

Comment:
I think you might be using Rust `OUTER_LINE_DOC` syntax here.

```suggestion
// existingFile implements fs.FileInfo / os.FileInfo and returns information
// as if it were a basic file that existed. This is used to mock out os.Stat.
```

Reply:
lol, indeed!

Reply:
I'll just re-ack if you push a changeset.